src/main/java/frc/robot/subsystems/drive/OdometryThreadRunner.java

```diff
@@ -122,7 +122,7 @@ public Class<State> getTypeClass() {
 
             @Override
             public String getTypeString() {
-                return "struct:Swerve.OdometryThreadRunner.State";
+                return "struct:OdometryThreadRunner.State";
```

Comment:
This fix is unrelated to this PR and should be included with it - it should get its own branch and have it PR'ed separately.

Reply:
Added back.